advocacy_docs/partner_docs/PreciselyConnectCDC/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Solution Summary'
+description: 'Explanation of the solution and its purpose'
+---
+
+Precisely Connect CDC (Change Data Capture) provides real-time data replication and change data capture functionality. Precisely Connect CDC real-time replication ensures that databases are in-sync for reporting, analytics, and data warehousing. You can replicate changes as they happen across hierarchical data stores (IMS, VSAM), relational databases, streaming frameworks, and the cloud.
+
+Precisely Connect CDC supports EDB Postgres Advanced Server and PostgreSQL databases in different modes for real-time data replication and change data capture functionality.
+
+<p align="center">
+  <img src="Images/ConnectCDCSolutionSummary.png">
+</p>
```

Comment:
Under the image I would add

EDB Postgres Advanced Server and PostgreSQL can be either the Source Database or Target Database in the diagram dependent on your configuration.

Reply:
Done, Guide updated